### DISCUSSION TITLE
Icarus

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#312] [Game request]
Icarus

Steam: https://store.steampowered.com/app/1149460/ICARUS/

info about dedicated servers: https://github.com/RocketWerkz/IcarusDedicatedServer/wiki

**Comment by Eurymachus** [comment]
I’m so out of touch I completely missed that Icarus has dedicated servers! This would be lovely, they just improved the way clients can connect/search too!

**Comment by mvaessen** [comment]
Created development item #473
